helpers/liquity.ts

```diff
@@ -63,3 +62,88 @@ export function liquityV2Exports(config: IJSON<LiquityV2Config>) {
   return { adapter: exportObject, version: 2 } as SimpleAdapter
 }
 
+const RedemptionEvent = 'event Redemption(uint _attemptedLUSDAmount, uint _actualLUSDAmount, uint _ETHSent, uint _ETHFee)'
+const BorrowingEvent = 'event LUSDBorrowingFeePaid(address indexed _borrower, uint _LUSDFee)'
+
+type LiquityV1Config = {
+  troveManager: string
+  holderRevenuePercentage?: number
+  protocolRevenuePercentage?: number
+  redemptionEvent?: string
+  borrowingEvent?: string
+}
+
+
+
+export const getLiquityV1LogAdapter: any = (config: LiquityV1Config): FetchV2 => {
+  const fetch: FetchV2 = async (fetchOptions) => {
+    const { createBalances, getLogs, api } = fetchOptions
+    const dailyFees = createBalances()
+
+    const redemptionEvent = config.redemptionEvent || RedemptionEvent
+    const borrowingEvent = config.borrowingEvent || BorrowingEvent
+
+    // Get brrower opertaor contract
+    const borrowerOperator = await api.call({ abi: 'address:borrowerOperationsAddress', target: config.troveManager })
+    
+    // redemtions fees
+    const redemptionLogs = await getLogs({
+      target: config.troveManager,
+      eventAbi: redemptionEvent,
+    })
+
+    // event LUSDBorrowingFeePaid(address indexed _borrower, uint _LUSDFee);
+    const borrowingLogs = await getLogs({
+      target: borrowerOperator,
+      eventAbi: borrowingEvent,
+    })
+
+    redemptionLogs.forEach((logs) => {
+      dailyFees.addGasToken(BigInt(logs[3]))
```

Comment:
would prefer this being `dailyFees.addGasToken(log._ETHFee)`

this way, even if this field is not the 4th parameter, it will work.

Reply:
same for borrow event